File: 코딩테스트 입문/개미 군단.js
function solution(hp) {
    let answer = 0;
    let [first,second,third] = [0,0,0];
    
    first = parseInt(hp/5);
    hp = hp - first*5;
    
    second = parseInt(hp/3);
    hp = hp - second*3;
    
    third = hp;
    
    answer = first + second + third;
    
    return answer;
}

File: 코딩테스트 입문/Python/개미 군단.py
def solution(hp):
    answer = 0    
    first, second, third = 0, 0, 0
    
    first = int(hp/5)
    hp = hp - first*5
    
    second = int(hp/3)
    hp = hp - second*3
    
    third = hp
    
    answer = first + second + third
    
    return answer